cabby/geo/walk_test.py

```diff
@@ -17,19 +17,20 @@
 import unittest
 from shapely.geometry.point import Point
 from cabby.geo import walk
+from cabby.geo.map_processing import map_structure
 
 
 class WalkTest(unittest.TestCase):
 
     def setUp(self):
         
         # Load map from disk.
-        self.map = map_structure.Map("Manhattan", 18, "/mnt/hackney/data/cabby/poi/")
+        self.map = map_structure.Map("Manhattan", 18, "/mnt/hackney/data/cabby/poi/v1/")
```

Comment:
We can't have hard-coded reference to data outside the repository. (Tests should be hermetic.)

Perhaps we can define a very small region (e.g. Chelsea in NYC) and then include the necessary data in a cabby/geo/testdata/ directory. (This will be useful beyond this.) WDYT?

Reply:
changed